glusterd2/xlator/validation.go

```diff
@@ -52,6 +53,34 @@ func validateBitrot(v *volume.Volinfo, key string, value string) error {
 	return nil
 }
 
+func validateQuota(v *volume.Volinfo, key string, value string) error {
+
+	// Check if quota is already enabled
+	if volume.IsQuotaEnabled(v) == false {
+		err := fmt.Errorf("Quota not enabled to set this value: '%s'", key)
+		return err
+	}
+
+	key = "quota." + key
```

Comment:
As the constants already have the quota prefix, key is gonna be `quota.quota.deem-statfs`. You'll have to remove this statement.

Reply:
done

Reply:
There was a confusion here. The key we get here is deem-statfs not quota.deem-statfs. having the quota prefix added is necessary!

Reply:
I see. In that case, **if** those constants you defined in `plugins/quota/consts.go` are not used anywhere else in gd2, use them directly here and remove that file altogether. This is along the lines of bitrot validation function.